### ISSUE TITLE
Accessibility: Automatically Decide (Or Suggest) Hyperparameters Values Without Plots

### ISSUE THREAD

**Issue by chigkim** [fastai/fastai#2072]
I'm visually impaired, so I can't look at the plots that Fastai provides to assist on deciding hyperparameters such as learning rate.
I know someone raised this question on one of the lesson videos as well, but probably not from accessibility angle.
It would be great if there is a features to suggest at least some potential values I should try.
This would be very useful for people who cannot utilize visualization tools as well as people who can.
Thanks!

**Comment by sgugger** [comment]
I hadn't thought of it that way. What would be an helpful way to suggest values then? I can think of two values to try (minimum/10 and steepest slope). Would printing them when you type learn.recorder.plot() be helpful enough?

Please let us know if there are other areas in which the library can be improved as we are committed to make it as accessible as possible.

**Comment by chigkim** [comment]
Thanks @sgugger for being open to the idea.
Yes, simply printing few suggested values would be enough for me.
I'll leave up to the experts to decide What values to print out. :)
As I go through the lessons, I'll let you know if I find other ways to improve accessibility. I suspect that mostly I would run into problems when I need to decide something looking at visualization tools like plots.
Since Jupyter is not accessible with screen reader, I just convert lesson notebookss to script using jupyter nbconvert, and run everything in headless terminal, and ignore matplotlib stuff.
Other things might not have big of impact, but learning rate seems pretty important from what I gathered so far.
Thanks again.

**Comment by tmabraham** [comment]
If you set `suggestion=True` as an argument for `learn.recorder.plot` it will print the minimum gradient. However, sometimes it is wrong if the plot is not so smooth.

**Comment by chigkim** [comment]
> If you set `suggestion=True` as an argument for `learn.recorder.plot` it will print the steepest slope. However, sometimes it is wrong if the plot is not so smooth.
Awesome, thanks @tmabraham! Is this implemented in the master branch now? I can just pull and start using it?

**Comment by sgugger** [comment]
Yes, this is even in v1.0.52. What I just added in master is to also print the minimum divided by 10 as another suggestion.
